c2rust-transpile/src/translator/mod.rs

```diff
@@ -4884,6 +4884,7 @@ impl<'c> Translation<'c> {
     }
 
     /// Convert a boolean expression to a boolean for use in && or || or if
+    #[allow(clippy::collapsible_match)]
```

Comment:
Is there a reason this one's skipped?  From a cursory glance, it seems like it'd be good.